integration-tests/quarkus/quarkus-jaxrs-undertow/src/test/resources/application.properties

```diff
@@ -0,0 +1 @@
+quarkus.class-loading.parent-first-artifacts=org.ow2.asm:asm
```

Comment:
Is this required because this is tested as a unit test, where ASM is not relocated?
If you try to attach the built agent separately, would it work without this setting?
If so, I think it implies we need better use a real integration test if possible.

Reply:
yes, this was needed because ASM is not relocated